### ISSUE TITLE
[SOLVED]Crystal Cluster Render Crash - Incompatibility with OptiFine HD v B6, Update to OptiFine HD v B7 to fix.

### ISSUE THREAD

**Issue by ScytheIronclaw** [Alec-WAM/CrystalMod#8]
Minecraft Version:  1.11.2
Forge Version:  13.20.0.2227
Mod Version:  1.11.2-4.3.0dev

Is this on a server yes/no:   no

Crash Log(s) if applicable (pastebin or git):  
[crash-2017-02-11_23.58.12-client.txt](https://github.com/Alec-WAM/CrystalMod/files/769087/crash-2017-02-11_23.58.12-client.txt)

Steps to reproduce:  
1.   Generate New World use the seed -1253876115710619987
2.   Go near x223 z387 y67
3.   Crash to desktop and optionally, rage.

Crash happened randomly when the following block generated in-world "Block: crystalmod:crystalcluster[type=green]"
Attempting to re-load the world after this crash happens results in the game crashaing every time the world loads. The chunk is effectively corrupted-breaking the world.

*Rolling back the the version before the version posted fixed the issue by removing the block in-world that was causing the issue.

**Comment by Alec-WAM** [comment]
It looks like better foliage is rendering the block and it is crashing due to a missing model. Could you try that again but without better foliage?

**Comment by ScytheIronclaw** [comment]
I removed better foliage and updated crystal mod, generated a completely new world and still encountered the same issue.

[crash-2017-02-22_01.30.30-client.txt](https://github.com/Alec-WAM/CrystalMod/files/792446/crash-2017-02-22_01.30.30-client.txt)


I created a separate instance using the exact same forge version, but only installed the Crystal Mod and nothing else.

My main instance has optifine and suffers these crashes when attempting to render the crystal clusters.
The debug instance I made did NOT have Optifine. So this mod is incompatible with Optifine.

There was an update to Optifine for my instance, after updating, and re-generate the same world and walking around, the issue no longer occurs. Issue resolved - **incompatibility with OptiFine v HD B6
Solution - Update to OptiFine 
HD B7.**

This issue can be closed.
